tests/www/test_utils.py

```diff
@@ -26,8 +27,11 @@
 
 class UtilsTest(unittest.TestCase):
 
-    def setUp(self):
-        super(UtilsTest, self).setUp()
+    @classmethod
+    def tearDownClass(cls):
+        import airflow.www.app
+        airflow.www.app.app = None
+        airflow.www.app.appbuilder = None
```

Comment:
Why do we do these in the `tearDownClass`?

All other changes look good to me.

Reply:
So `cached_app`/`cached_appbuilder` are "memoized" meaning they will return the previous value of these variables if set, and otherwise construct a new one.

Since we passed custom config we want to "remove" these cached variables so other tests that might call cached_app etc _don't_ pick these up.

I'll add this reason as a comment 👍

Reply:
Oh actually in this test we don't need it. I was confusing the three PRs I have open at the same time.

I can remove this here.